scripts/entrypoint.pl

```diff
@@ -37,7 +37,14 @@
 sub cmd_httpd  {
     check_data_dir();
     wait_for_db();
-    run( '/usr/sbin/httpd', '-DFOREGROUND', '-f', '/app/httpd/httpd.conf' );
+    my @httpd_args = (
+        '-DFOREGROUND',
+        '-f' => '/app/httpd/httpd.conf',
+    );
+    if ($ENV{BMO_inbound_proxies} eq '*' && $ENV{BMO_urlbase} =~ /^https/sm) {
+        unshift @httpd_args, '-DHTTPS';
+    }
```

Comment:
this deserves a comment to explain why this is required.

pretty sure that regex doesn't needs /sm

Reply:
without /sm perlcritic is mad. ;)